app/controllers/admin/billboards_controller.rb

```diff
@@ -27,7 +27,7 @@ def create
 
       if @display_ad.save
         flash[:success] = I18n.t("admin.display_ads_controller.created")
```

Comment:
Should we maybe also change the I18n keys that refer to the old controller explicitly? (I don't _think_ we're using any "automatic" keys, but still, it seems a little odd to keep the old keys around?)